### DISCUSSION TITLE
How to enable "Reaction" in CallChatComposite

### DISCUSSION THREAD

**Discussion by lovedota** [Azure/communication-ui-library#4420] [Q&A]
I am following this guide https://azure.github.io/communication-ui-library/?path=/docs/addingreactions--page but cannot see the Reaction Button Enable.

**Comment by alkwa-msft** [comment]
love the level of engagement! 

This page is foreshadowing a feature in our release next week. Stay tuned!

**Comment by lovedota** [comment]
Is it released somewhere @alkwa-msft ?
